src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/SpecifyIFormatProviderTests.cs

```diff
@@ -945,6 +945,45 @@ End Class
 ");
         }
 
+        [Fact]
+        [WorkItem(5999, "https://github.com/dotnet/roslyn-analyzers/issues/5999")]
+        public async Task CA1305_GuidParse_NoDiagnosticsAsync()
+        {
+            await new VerifyCS.Test
+            {
+                ReferenceAssemblies = new ReferenceAssemblies(""), // workaround for lack of .NET 7 Preview 4 reference assemblies
```

Comment:
@sharwell Don't we have a way to enable .NET 7 preview on tests?

Reply:
It may have been the case that only an earlier preview (not Preview 4) was available when I wrote this test?

Reply:
Fixing the test is tracked in https://github.com/dotnet/roslyn-analyzers/issues/6052